src/probnum/random_variables/_utils.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import scipy.sparse
+
+from . import _random_variable
+from . import _dirac
+from . import _scipy_stats
+
+
+def asrandvar(obj) -> _random_variable.RandomVariable:
```

Comment:
I assume you are type hinting `_random_variable.RandomVariable` instead of `probnum.RandomVariable` to avoid circular dependencies. From a quick local test this might break links in the automatic documentation of types with Sphinx, i.e. you get the type `RandomVariable` but it does not actually link to the `RandomVariable` class. I'm not sure whether this is solvable with forward references.

Reply:
My local test actually links this correctly, that's weird! How should we go about this?

Reply:
hmm strange. Since this is a doc issue i can fix it when going through the docs.